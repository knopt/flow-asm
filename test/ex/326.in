3
12
-8.623276930563598
-9.312416215727987 4.069283415613487 9.613599118631655 
1.160329258114018 -0.13266905945622476 -0.21236382884839244 
9.493046709466856 -0.3449709681140334 -8.519465209654921 
6.43987050821416 -6.194770009543276 0.4821605119192718 
-0.5968346540628229 2.4976774138776747 7.053447584729614 
-0.22054335197580244 -7.279587398155726 -7.5115174521753385 
-1.2939437004433163 -2.0745852879241866 1.6495368933557657 
-2.6277111212517656 1.2545198538160829 -1.2396116490835318 
-5.29971648463575 9.672674146014785 -5.287371100196139 
-6.774155342270552 -5.3507940429040435 -6.080903537844451 
-9.517698991943115 -4.1039518585033985 -4.681513497505698 
-5.618318234685315 2.3891455723539465 -8.236479877869257 

-3.0135407275156894 9.4929712493459 3.02830269926889 9.996617109489975 4.920497008280616 -6.8787777701873125 -3.5805574939080937 2.44031624686796 6.74699246886826 -4.982623781522881 2.583129425597381 5.954296024029556 
