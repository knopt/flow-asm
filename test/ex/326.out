-316.10281224384926 180.78270225831733 226.20053796099842 
-32.355911442194746 -49.53212416219858 -49.77547962697179 
159.08226236305617 -108.05541057328435 -320.6364961942027 
52.641005420222385 -424.10152738484874 68.76888805446087 
-149.27796794901073 177.89730303098565 466.4436496809274 
54.33806617032626 -306.59100657557843 -347.3037810181494 
36.62739078051804 -3.848054644110256 218.08056319560157 
-107.41472528626228 69.34289451324611 -99.65562901871262 
-188.98759540142981 588.8225410425999 -218.2197086313327 
-208.5662459107991 -97.8882280925598 -184.18228254165524 
-343.6767774468087 33.425081804204375 -22.140059780453196 
-142.5100900767684 230.10751882244193 -166.15558380661201 
